packages/blade/tsconfig-generate-types.native.json

```diff
@@ -9,7 +9,8 @@
     "**/**.stories.tsx",
     "**/**.stories.internal.tsx",
     "**/**.web.ts",
-    "**/**.web.tsx"
+    "**/**.web.tsx",
+    "**/ThemePlayground/**.tsx"
```

Comment:
ThemePlayground import from stories, stories are already ignored but because we don't ignore ThemePlayground, it starts typechecking stories too